packages/edit-post/src/components/header/fullscreen-mode-close/style.scss

```diff
@@ -14,9 +14,10 @@
 		background: #23282e; // WP-admin gray.
 		color: $white;
 		border-radius: 0;
-		height: $header-height;
+		height: $header-height + 1px;
```

Comment:
This small hack is here to not show the light border under the toggle, especially in "preview" or "template mode"

Reply:
You can use $border-width here, that returns 1px. Not because we need to change the contents of that variable very often, but it helps work sort of like a comment to explain what that pixel is for.

Reply:
We should probably add a similar hack to "hide" the inspector border too.